ilp-core/src/main/java/org/interledger/core/InterledgerProtocolException.java

```diff
@@ -35,20 +35,20 @@
    * Required-args constructor.
    *
    * @param interledgerRejectPacket An instance of {@link InterledgerRejectPacket} that is the
-   *                                 underlying error encapsulated by this exception.
+   *                                underlying error encapsulated by this exception.
    */
   public InterledgerProtocolException(final InterledgerRejectPacket interledgerRejectPacket) {
-    super("Interledger Rejection.");
+    super(String.format("Interledger Rejection: %s", interledgerRejectPacket.getMessage()));
     this.interledgerRejectPacket = fillInterledgerRejectPacket(interledgerRejectPacket);
   }
 
   /**
-   * Constructs a new Interledger protocol exception with the specified reject packet and
-   * detail message.
+   * Constructs a new Interledger protocol exception with the specified reject packet and detail
+   * message.
    *
    * @param interledgerRejectPacket An instance of {@link InterledgerRejectPacket} that is the
-   *                                 underlying error encapsulated by this exception.
-   * @param message The detail message.
+   *                                underlying error encapsulated by this exception.
+   * @param message                 The detail message.
```

Comment:
infi-nit:
```
@param message                 The detail message.
```

Reply:
I think the spacing is purposeful -- it aligns with the description of the `message` field.